versions.props

```diff
@@ -15,7 +15,8 @@ com.palantir.config.crypto:encrypted-config-value-module = 1.0.0
 com.palantir.docker.compose:docker-compose-rule* = 0.31.0
 com.palantir.docker.proxy:docker-proxy-rule = 0.3.0
 com.palantir.remoting2:* = 2.3.0
```

Comment:
Do you still need to lock remoting2?

Reply:
I'm afraid so (though I somehow doubt that there will be consequences if we exclude it). 

We still return remoting2 style errors, because remoting3 Jersey Servers doesn't quite support the old error format (its legacy support only seems to extend to specific classes of exceptions). We need TimeLock to be compatible with both older and newer versions of Atlas.